pkg/ride/converters.go

```diff
@@ -871,6 +873,10 @@ func invokeScriptWithProofsToObject(scheme byte, tx *proto.InvokeScriptWithProof
 	return r, nil
 }
 
+func ethereumInvokeScriptWithProofsToObject(scheme byte, tx *proto.EthereumTransaction) (rideObject, error) {
+	return nil, nil
+}
+
```

Comment:
Implement this.

Reply:
There is no need. The transaction object should be nil

Reply:
Almost done. I'm waiting for ethabi package to be fixed as it's necessary here